archetypes/blog.md

```diff
@@ -1,12 +1,13 @@
 ---
 title: {{ replaceRE "[-_]" " " .Name | title }}
-linkTitle: ADD A SHORT TITLE HERE # TODO: add short title or remove this line
-date: {{ dateFormat "2006-01-02" .Date }}
+linkTitle: ADD A SHORT TITLE HERE # Mandatory, make sure that your short title. 
+date: {{ dateFormat "2006-01-02" .Date }} # Put the current date, we will keep the date updated until your PR is merged
 author: >- # If you have only one author, then add the single name on this line in quotes.
-  [Author1 Name](https://github.com/author1_GH_ID),
+  [Author1 Name](https://github.com/author1_GH_ID) (Organisation Name 1),
```

Comment:
We'll need to have a conversation about `o`/ou` and `z`/s` some day, but today is not that day

Reply:
Grow up in a country where "th" is pronounced as "z", your teacher forces to write British English while everything you read and watch is American English and you know where this is coming from 🤣